### PR TITLE
DOC: Add missing parameters to Series constructor

### PR DESCRIPTION
**Pull request by SHuangISU**
Added documentation to pandas.Series for "name" and "fastpath" parameters.

### CODE REVIEW

**Comment by sbrice** [comment]
1. You should install this tool on your machine: https://github.com/tj/git-extras
2. Then make the five changes as instructed by the pep8speaks comment above
3. Use `git squash HEAD~1` to undo the last commit: https://github.com/tj/git-extras/blob/master/Commands.md#git-squash
4. Do `git commit -am "DOC: add API docstrings to pandas.Series for 'name' and 'fastpath'"` as per the guidelines: https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#committing-your-code
5. To override your first commit you'll have to do `git push -f` that should automatically updat e this PR

**Comment by jbrockmendel** [comment]
for what its worth, when merging a PR we squash all commits into one, so its really only step 2 that is needed

File: pandas/core/series.py [hunk 1]
```diff
@@ -170,8 +170,14 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
         Data type for the output Series. If not specified, this will be
         inferred from `data`.
         See the :ref:`user guide <basics.dtypes>` for more usages.
+    name : str
+        This is the variable name of the series. If it in a DataFrame, the column 
+        will be named according to this name parameter.
```
Comment (jbrockmendel) [inline pandas/core/series.py]: is the DataFrame sentence relevant?  i cant think of a case where that comes up.

The first sentence doesn't need to be a complete sentence: "Name of the Series." is fine

File: pandas/core/series.py [hunk 2]
```diff
@@ -170,8 +170,14 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
...
         inferred from `data`.
         See the :ref:`user guide <basics.dtypes>` for more usages.
+    name : str
+        This is the variable name of the series. If it in a DataFrame, the column 
+        will be named according to this name parameter. 
+        
     copy : bool, default False
         Copy input data.
+        
+    fastpath: internal paraemter
```
Comment (jbrockmendel) [inline pandas/core/series.py]: no newline before this, type it as bool, paraemter--> parameter (on the next line)

**Comment by datapythonista** [comment]
@SHuangISU can you address the review comments please? We've got another PR adding these parameters to this doc, if you're not planning to finish this PR, please let me know, so I'll keep the other one open. Thanks!

**Comment by sbrice** [comment]
@datapythonista - I'd say you could give @SHuangISU 24hrs to respond and if he doesn't get to it you can close this and perhaps point out a similar issue that's available

**Comment by datapythonista** [comment]
Thanks for the suggestion @sbrice, but contributors to pandas are volunteers working in their scarce free time, and giving 24h to respond sounds unreasonable and rude to me.

**Comment by sbrice** [comment]
Yeah, the suggestion was within the context of our event i wasn't sure who the other PR was coming from

**Comment by datapythonista** [comment]
Ah, that makes sense, didn't get that. Thanks!

**Pull request closed without merging**